content/release-10-16-0/devices-10-16-0-bundle/10-16-0-200.md

```diff
@@ -0,0 +1,65 @@
+---
+weight: 37
+title: Release 10.16.0.200
+layout: bundle
+---
+
+<!--10.16.0.164-10.16.0.200-->
+
+### Fixes
+
+<div><table ><colgroup>
+<col style="width: 15%;"><col style="width: 70%;"><col style="width: 15%;"></colgroup>
+<thead><tr>
+<th>
+Component</th>
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Connectivity</td>
+<td>Due to a compatibility error of the connectivity microservice with the Kite provider, an error occurred on the metrics page, when the monthly consumption data was an out-of-scope integer. This has been fixed by changing the data type.</td>
```

Comment:
is it really two separate errors, or is the compatibility error showing up in the metrics page? then we should revise the entry and talk about a single error being raised

Reply:
Don´t get that. It is about one error. There is a compatibility error. And this leads to an error in the metric page.

Reply:
How about that?

Reply:
purrfect